spec/unit/inflecto/class_methods/uncountable_predicate_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+describe Inflecto, '.uncountable?' do
+  it 'truthy when empty string' do
+    Inflecto.uncountable?('').should == true
```

Comment:
We should use the most strict matcher here. `Object#==` is more permissive than `Object#eql?` than `Object#equal?`. I think `should be(true)` is the strictest and less lowerfullest primitive to use here.